backend/src/main/java/gov/cdc/usds/simplereport/service/ReminderService.java

```diff
@@ -32,19 +33,25 @@ public ReminderService(OrganizationQueueRepository orgQueueRepo, EmailService em
     _emailService = emailService;
   }
 
+  /**
+   * Wrapper method for sending account reminder emails so automation can call the inner method
+   * without hitting the lock or conditions.
+   */
+  @Scheduled(cron = "0 0 1 * * *", zone = "America/New_York")
+  @SchedulerLock(
+      name = "ReminderService_sendAccountReminderEmails",
+      lockAtLeastFor = "PT30S",
```

Comment:
two questions
what is the PT in front of the time? the [docs](https://github.com/lukas-krecan/ShedLock#:~:text=%40SchedulerLock(name%20%3D%20%22scheduledTaskName%22%2C%20lockAtMostFor%20%3D%20%2214m%22%2C%20lockAtLeastFor%20%3D%20%2214m%22)) just have numbers